### PR TITLE
correcting configuration error

### PR DESCRIPTION
**Pull request by s-santillan**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs

Addresses the following:
* Pablo's suggested renames of labels in sample ci configs
* Sebas's fix to bitbucket
* Bence's feedback
* Andy and Raj for SSC supported languages

For Pablo: Your [suggestion](https://github.com/returntocorp/semgrep-docs/pull/878/files#r1105258239) pointed out that CI could be implied and that it's redundant. I did try "scan" but that also seemed redundant. I think the snippets are clear even without the word (CI or scan) so I did away with it. :)

### CODE REVIEW

**Comment by s-santillan** [comment]
For Bence: 
feature

File: src/components/reference/_scm-feature-reference.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Features dependent our CI and SCM:
```
Comment (underyx) [inline src/components/reference/_scm-feature-reference.md:1]:
```suggestion
Features dependent on CI and SCM:
```

File: src/components/reference/_scm-feature-reference.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+| Feature                | GitHub with GitHub Actions | GitLab with GL CI/CD | GitHub or GitLab with other CI providers |
+| -------                | -------------------------- | ------------------   | -----------------------                  |
+| **Diff-aware scanning**    | ✅                         | ✅                  | ✅  (May need additional set up) |
+| **Hyperlinks**             | ✅                         | ✅                  | ✅  (May need additional set up) |
+| **PR or MR comments**      | ✅                         | ✅                  | ✅  (May need additional set up) |
+| **SCM security dashboard** | ✅ GitHub Advanced Security Dashboard | ✅ GitLab Security Dashboard | ❌ No |
```
Comment (underyx) [inline src/components/reference/_scm-feature-reference.md:8]: love this! 😻

File: src/components/reference/_scm-feature-reference.md (hunk 1 above)
Comment (underyx) [reply src/components/reference/_scm-feature-reference.md:1]:
```suggestion
Integrations with source code providers, dependent on CI provider:
```

another wording that might be nice

File: src/components/reference/_scm-feature-reference.md (hunk 2 above)
Comment (s-santillan) [reply src/components/reference/_scm-feature-reference.md:8]: Thanks for the feedback Bence :)

**Comment by s-santillan** [comment]
@andy-r2c - Can you please let me know when we've been doing rule backfill coverage for the languages above? Thank you.

**Comment by s-santillan** [comment]
Latest update includes Matthew's feedback:

**Review by andy-r2c — APPROVED** [review]

**Review by underyx — APPROVED** [review]

**Pull request merged**